saltcloud/libcloudfuncs.py

```
@@ -103,6 +104,7 @@ def get_size(conn, vm_):
             return size
         if str(size.name) == str(vm_['size']):
             return size
+    raise ValueError
```

These `raise` calls should have a helpful error message, such as `raise ValueError("The foobar is missing baz")`.